src/tracked-models/creators.ts

```diff
@@ -31,8 +31,12 @@ export const createContributorNode = (
   const { tr } = state
   doc.descendants((node, pos) => {
     if (node.type === schema.nodes.contributors) {
+      let insertPos = pos + node.nodeSize - 1
```

Comment:
Can we insert the node at the beginning instead? The display should not be affected since it uses the `priority` for sorting.

Reply:
yes we can.